subsys/spm/Kconfig

```diff
@@ -148,16 +160,52 @@ config SPM_NRF_EGU2_NS
 	bool "EGU2 is Non-Secure"
 	default y
 
+config SPM_NRF_EGU3_NS
+	bool "EGU3 is Non-Secure"
+	default y
+
+config SPM_NRF_EGU4_NS
+	bool "EGU4 is Non-Secure"
+	default y
+
+config SPM_NRF_EGU5_NS
+	bool "EGU4 is Non-Secure"
+	default y
+
 config SPM_NRF_FPU_NS
 	bool "FPU is Non-Secure"
 	default y
 
+config SPM_NRF_TWIM0_NS
+	bool "TWIM0 is Non-Secure"
+	default n
+
+config SPM_NRF_TWIM1_NS
+	bool "TWIM1 is Non-Secure"
+	default y
+
 config SPM_NRF_TWIM2_NS
 	bool "TWIM2 is Non-Secure"
 	default y
 
+config SPM_NRF_TWIM3_NS
+	bool "TWIM3 is Non-Secure"
+	default y
+
+config SPM_NRF_SPIM0_NS
+	bool "SPI0 is Non-Secure"
+	default n
+
+config SPM_NRF_SPIM1_NS
+	bool "SPI1 is Non-Secure"
+	default y
+
+config SPM_NRF_SPIM2_NS
+	bool "SPI2 is Non-Secure"
+	default y
+
 config SPM_NRF_SPIM3_NS
-	bool "SPIM3 is Non-Secure"
+	bool "SPI3 is Non-Secure"
```

Comment:
Why this change?

Reply:
because SPIM and SPIS have the same address and aren't separately configurable secure/non-secure at least that's how I read the product specification.

Reply:
I see. We should be more explicit about that in the printout or the Kconfig text. Or maybe `depends on` or `select` when two configs are linked like that.

Reply:
So you want multiple configs pr SPI/I2C and other M/S ports?

Reply:
Wait, is there no `SPM_NRF_SPI3_NS`? If so, why rename only the prompt and not the config as well? Mismatched name/prompt is confusing.

Anyway, I meant that if there are `SPM_NRF_*` configs which must be set to the same because of HW restrictions, that should be reflected in the Kconfig text, or (even better) enforced via `depends on` or `select`